apps/event/models.py

```diff
@@ -423,9 +425,9 @@ def transformer(datum):
             return {
                 **datum,
                 **dict(
-                    start_date_accuracy=getattr(DATE_ACCURACY.get(datum['start_date_accuracy']), 'name', ''),
-                    end_date_accuracy=getattr(DATE_ACCURACY.get(datum['end_date_accuracy']), 'name', ''),
-                    event_type=getattr(Crisis.CRISIS_TYPE.get(datum['event_type']), 'name', ''),
+                    start_date_accuracy=get_enum_label(datum, 'start_date_accuracy', DATE_ACCURACY),
+                    end_date_accuracy=get_enum_label(datum, 'end_date_accuracy', DATE_ACCURACY),
+                    event_type=get_enum_label(datum, 'event_type', Crisis.CRISIS_TYPE),
```

Comment:
I think we can just use it directly
```suggestion
                    start_date_accuracy=DATE_ACCURACY.get(datum['start_date_accuracy']).label,
                    end_date_accuracy=DATE_ACCURACY.get(datum['end_date_accuracy']).label,
                    event_type=Crisis.CRISIS_TYPE.get(datum['event_type']).label,
```

Reply:
`start_date_accuracy=DATE_ACCURACY.get(datum['start_date_accuracy']).label` gives error when `DATE_ACCURACY.get(datum['start_date_accuracy']` is None . So May be we need to write function to accept None values rather writing  `start_date_accuracy=DATE_ACCURACY.get(datum['start_date_accuracy']).label if datum['start_date_accuracy'] else None`  @thenav56

Reply:
Sounds good. let's do something like this instead https://github.com/idmc-labs/helix-server/pull/520#discussion_r1346980760